core-service/src/main/java/com/crmfoodestablishment/coreservice/entity/Order.java

```diff
@@ -0,0 +1,40 @@
+package com.crmfoodestablishment.coreservice.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.type.SqlTypes;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false, unique = true)
+    private Integer id;
+
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "uuid", nullable = false)
+    private UUID uuid;
+
+    @JdbcTypeCode(SqlTypes.BINARY)
+    @Column (name = "user_uuid", nullable = false)
+    private UUID userUuid;
+
+    @Column(name = "creation_date", nullable = false)
+    private LocalDateTime creationDate;
+
+    @Column(name = "comment", length = 128)
+    private String comment;
+
+    @Column(name = "total_price", nullable = false)
+    private Double totalPrice;
```

Comment:
it seems to me that it will be enough to keep total_price only in order
in dish_order we can remove it. 
+ let's calculate price here (use @PrePersist)

**example:**
@PrePersist
    public void sumOrder() {
        sum = vines.stream()
            .mapToDouble(vine -> vine.getVine().getPrice() * vine.getVineAmount()
        ).sum();
    }